activiti-core/activiti-process-validation/src/main/java/org/activiti/validation/validator/impl/EventValidator.java

```diff
@@ -53,8 +55,9 @@ protected void executeValidation(BpmnModel bpmnModel, Process process, List<Vali
             handleTimerEventDefinition(process, event, eventDefinition, errors);
           } else if (eventDefinition instanceof CompensateEventDefinition) {
             handleCompensationEventDefinition(bpmnModel, process, event, eventDefinition, errors);
+          } else if (eventDefinition instanceof LinkEventDefinition linkEventDefinition) {
+              handleLinkEventDefinition(process, event, linkEventDefinition, errors);
           }
```

Comment:
As discussed together this will be a good candidate for a refactoring in the future and apply strategy pattern. But it's out of scope of the current PR.